assemblies/installing-odh-v2.adoc

```diff
@@ -22,6 +22,15 @@ Installing Open Data Hub involves the following tasks:
 Version 2 of the Open Data Hub Operator represents an alpha release, accessible only on the *fast* channel. Later releases will change to the *rolling* channel when the Operator is more stable.
 --
 
+[IMPORTANT]
+====
+Data Science Pipelines (DSP) 2.0 contains an installation of Argo Workflows. {productname-short} does not support direct customer usage of this installation of Argo Workflows.
+
+If there is an existing installation of Argo Workflows that is not installed by DSP on your cluster, installations of and upgrades to {productname-short} 2.10.0 with DSP will fail.
+
+To install or upgrade to {productname-short} 2.10.0 with DSP, you must ensure that no separate installation of Argo Workflows exists on your cluster.
```

Comment:
the words "you must" can probably be omitted.

Reply:
Good spot, will update.